screens/Landmark.js

```diff
@@ -116,18 +149,15 @@ class LandmarkScreen extends React.Component {
               })}
             </ScrollView>
           )}
-        </ScrollView>
-      </View>
+        </View>
+      </ScrollView>
     );
   }
 }
 
 const mapDispatchToProps = dispatch => ({
   fetchDirections: coordinate => dispatch(getDirections(coordinate)),
-  clearDirections: () => dispatch(clearDirections())
+  eraseDirections: () => dispatch(clearDirections())
```

Comment:
just curious why you changed this method name?

Reply:
It felt a little confusing since the method being imported from the store was also clearDirections.